### ISSUE TITLE
AssertionError in gated_cnn.py

### ISSUE THREAD

**Issue by Mahhos** [yangperasd/gated_cnn#1]
Hi. When I run the code, I get below error. Do you have any idea why this is happening?
The only change I have made to the code is that I modify `from keras.layers import Wrapper` to `from tensorflow.keras.layers import Wrapper`. Because it could not detect Wrappers with previous code. 
`  File "/Users/admin/Downloads/gated_cnn-master/py/mnist_gated_cnn.py", line 59, in <module>
    model.add(GatedConvBlock(Convolution2D(nb_filters*2, kernel_size, padding='same')))
  File "/Users/admin/Downloads/gated_cnn-master/py/gated_cnn.py", line 26, in __init__
    super(GatedConvBlock, self).__init__(conv_layer,**kwargs)
  File "/Users/admin/opt/anaconda3/envs/sentiment/lib/python3.6/site-packages/tensorflow_core/python/keras/layers/wrappers.py", line 52, in __init__
    assert isinstance(layer, Layer)
AssertionError`

**Comment by yangperasd** [comment]
tensorflow 1.0.0 does not have keras wrapper。
when I tested this code, I used tensorflow 1.0.0。
Or you can use `from keras.layers import Wrapper` with keras 2.1.2

**Comment by Mahhos** [comment]
Thank you.
